### ISSUE TITLE
Finetune VAE

### ISSUE THREAD

**Issue by vanche** [hpcaitech/Open-Sora#743]
I'm interested in fine-tuning your VAE, but I'm getting very high loss values. I'm wondering if it's training properly. What was the range of loss values when you were training?
